db/script.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+$request = curl_init('https://dev.maydenacademy.co.uk/resources/store_products/products.json');
+curl_setopt($request, CURLOPT_RETURNTRANSFER, 1);
+$response = curl_exec($request);
+$data = json_decode($response, true);
+curl_close($request);
+
+$db = new PDO('mysql:host=db;dbname=robot_stores', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('DROP TABLE `products`');
+$query->execute();
+$query = $db->prepare('DROP TABLE `categories`');
+$query->execute();
+$query = $db->prepare('DROP TABLE `characters`');
+$query->execute();
+
+$query = $db->prepare('CREATE TABLE `products` (
+`id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `title` varchar(255) NOT NULL DEFAULT \'\',
+  `price` float NOT NULL,
+  `image` varchar(255) NOT NULL DEFAULT \'\',
+  `category_id` int(11) NOT NULL,
+  `character_id` int(11) NOT NULL,
+  `description` text NOT NULL,
+  `image2` varchar(255) DEFAULT \'\',
+  `image3` varchar(255) DEFAULT \'\',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=0 DEFAULT CHARSET=latin1');
+$query->execute();
+
+$query = $db->prepare('CREATE TABLE `characters` (
+`id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL DEFAULT \'\',
+  `image` varchar(255) NOT NULL DEFAULT \'\',
+  `description` text NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=0 DEFAULT CHARSET=latin1');
+$query->execute();
+
+$query = $db->prepare('CREATE TABLE `categories` (
+`id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL DEFAULT \'\',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=0 DEFAULT CHARSET=latin1');
+$query->execute();
+
+$query = $db->prepare('INSERT INTO `products` (`title`, `price`, `image`, `category_id`, `character_id`, `description`, `image2`, `image3`) VALUES (:title, :price, :image, :category_id, :character_id, :description, :image2, :image3)');
+foreach ($data['products'] as $product) {
+    if (!isset($product['image2'])){
```

Comment:
Please add a space between the closing rounded bracket and the opening curly bracket :) Also please do this for line 54 :)